images/dotnet/README.md

```diff
@@ -26,4 +26,67 @@ docker pull cgr.dev/chainguard/dotnet:latest
 <!--getting:end-->
 
 <!--body:start-->
+## Variants
+We have two images for both sdk and runtime available: a `dotnet-sdk:latest-dev` and a `dotnet-runtime:latest-dev` variant that contains `dotnet` and a shell.
+
+These images are available on `cgr.dev`:
+
+```
+docker pull cgr.dev/chainguard/dotnet-sdk:latest
+docker pull cgr.dev/chainguard/dotnet-sdk:latest-dev
+
+docker pull cgr.dev/chainguard/dotnet-runtime:latest
+docker pull cgr.dev/chainguard/dotnet-runtime:latest-dev
+```
```

Comment:
```suggestion
```

There's a section in `metadata.yaml` called `compatibility_notes` that would be a better spot for this. It can use inline markdown like this:

```
compatibility_notes: |
    The Chainguard .NET images are available on `cgr.dev` as two variants: `dotnet-sdk` and `dotnet-runtime`. The SDK variant contains additional tooling to facilitate development and building, while the runtime variant contains only the runtime to execute .NET applications. Both the `sdk` and `runtime` images also have `latest-dev` version that contain a shell and various other tools for development.

    ### SDK
    ```
    docker pull cgr.dev/chainguard/dotnet-sdk:latest
    ```

    ### Runtime
    ```
    docker pull cgr.dev/chainguard/dotnet-runtime:latest
    ```
```

Reply:
Is there a command I can use to validate this command and see it work? I have been trying to use monopod to verify this but it keeps saying it is trying to find "matadata.hcl" instead of the .yaml file mentioned.

Reply:
I think you might have an out of date `monopod` - In your image's repository checkout, run `cd monopod; go install; cd ..; monopod readme` and I think that will fix things up.